### PR TITLE
Australicus Slime Mutator buyable with TC once more

### PR DESCRIPTION
**Pull request by Elli-Skala**
spooders

## About The Pull Request

removes a skyrat edit removing the Australicus Slime Mutator from the sci, robotics, and RD uplink

## Why It's Good For The Game

enables brilliant new traitor interaction with spider friends
## Changelog

:cl:
balance: Australicus Slime Mutator reported to be once more circulating on the syndicate black market
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — APPROVED** [review]
This can only be good

**Comment by ReturnToZender** [comment]
Update your branch using the button on Github and it'll give you the fabled green checkmark

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Add Bubber edit comments

File: code/modules/uplink/uplink_items/job.dm [hunk 1]
```diff
@@ -310,7 +310,6 @@
 	restricted_roles = list(JOB_ROBOTICIST, JOB_RESEARCH_DIRECTOR)
 	surplus = 15
 
-/* //SKYRAT REMOVAL START
```
Comment (projectkepler-ru) [inline code/modules/uplink/uplink_items/job.dm:313]: //Bubber Edits

File: code/modules/uplink/uplink_items/job.dm [hunk 2]
```diff
@@ -323,8 +322,6 @@
 	restricted_roles = list(JOB_RESEARCH_DIRECTOR, JOB_SCIENTIST, JOB_ROBOTICIST)
 	surplus = 10
 
-*/
-// SKYRAT EDIT END
```
Comment (projectkepler-ru) [inline code/modules/uplink/uplink_items/job.dm:327]: //Bubber Edits end

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]

**Pull request closed without merging**